### PR TITLE
[ Fabric ] Add new channel to an existing hyperledger fabric network

### PR DESCRIPTION
**Pull request by sivaramsk**
**Changelog**
- Add new channel to the existing network.

**Reviewed by**
@jagpreetsinghsasan @lakshyakumar 

**Linked issue**
#1030

### CODE REVIEW

**Comment by sivaramsk** [comment]
I think I made a mistake in the PR while adding a new md file, I did not sign it and when I amended it, I did not force push it. Should I create a new PR?

**Comment by sownak** [comment]
@sivaramsk you can do a reset on your branch and push --force

File: platforms/hyperledger-fabric/configuration/add-new-channel.yaml [hunk 1]
```diff
@@ -0,0 +1,112 @@
+# This playbook is a subsequent playbook
+##########################################################################################
+# DO NOT RUN THIS IF YOU HAVE NOT RUN deploy-network.yaml and deployed the Fabric network
+# and want to add a new channel to the existing network
+##########################################################################################
+# This playbook can only be run after all pods and orderers are available
+# Please use the same network.yaml to run this playbook as used for deploy-network.yaml
```
Comment (sownak) [inline platforms/hyperledger-fabric/configuration/add-new-channel.yaml:7]: Dont think we will use the "same" network.yaml, surely new channels should be there.

Comment (sivaramsk) [reply platforms/hyperledger-fabric/configuration/add-new-channel.yaml]: Fixed it.

File: platforms/hyperledger-fabric/configuration/add-new-channel.yaml [hunk 2]
```diff
@@ -0,0 +1,113 @@
...
+
+    # delete build directory
+    - name: Remove build directory
+      file:
+        path: "./build"
+        state: absent
+
+    # Create or get Orderer crypto materials
+    - include_role:
+        name: "create/crypto/orderer"
```
Comment (jagpreetsinghsasan) [inline platforms/hyperledger-fabric/configuration/add-new-channel.yaml:23]: Do we require the crypto role (orderer & peer) for adding a new channel?

**Review by jagpreetsinghsasan — CHANGES_REQUESTED** [review]

File: platforms/hyperledger-fabric/configuration/add-new-channel.yaml (hunk 2 above)
Comment (sivaramsk) [reply platforms/hyperledger-fabric/configuration/add-new-channel.yaml:23]: Yes, channel creation does need the crypto material of both orderers and the peer as far as I understand the steps.

**Review by sownak — APPROVED** [review]
LGTM

**Review by jagpreetsinghsasan — APPROVED** [review]
Looks fine

**Review by lakshyakumar — APPROVED** [review]
Looks good

**Pull request merged**